recipes-crosswalk/crosswalk/crosswalk_20.50.533.12.bb

```diff
@@ -419,6 +448,7 @@ DEFAULT_CONFIGURATION = "\
 
 # Not available on Yocto, so we use
 # what comes bundled with Chromium.
+# -Duse_system_flac=1
```

Comment:
It seems that GN will work as expected, though I haven't had a try yet.

I agree not to use system flac, but just not satisfied with the reason associated with this change.

In fact, there are other system libs that are unsafe to enable for the same reason (e.g. harfbuzz).